src/www/components/popup/TeamPopup.jsx

```diff
@@ -34,6 +35,21 @@ class TeamPopup extends Component {
       .catch((err) => this.handleError(err.response.data));
   };
 
+  updateTeam = (name, description, categories, photo) => {
+    const fd = new FormData();
+    console.log(this.props.action)
+    fd.set("name", name);
+    fd.set("description", description);
+    fd.set("categories", JSON.stringify(categories));
+    if (photo) {
+      fd.set("photo", photo);
+    }
+    axios
+      .patch(this.props.action, fd)
+      .then((res) => window.location.reload(false))
```

Comment:
I've found that this sometimes causes the page to not be refreshed properly.
Removing `false` from reload's arguments seems to be the fix for me.
```suggestion
      .then(() => window.location.reload())
```